circle.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
     - nvm install 6
     - nvm install 8
     - rm -rf node_modules
+
```

Comment:
Was this line added for a reason? Seems unnecessary and inconsistent with the style.